test/shared/base/view.test.js

```diff
@@ -310,6 +311,30 @@ describe('BaseView', function() {
         });
       });
 
+      context('contains an array of model data to build a collection', function () {
+        var modelUtilsMock;
+
+        beforeEach(function() {
+          collection = new MyCollection([], { app: this.app, params: { test: 'test' } });
+          modelUtilsMock = sinon.mock(modelUtils);
+          modelUtilsMock.expects("getCollection").withArgs('MyCollection', [], { parse: true, app: this.app, params: params }).returns(collection);
+        });
+
+        afterEach(function() {
+          modelUtilsMock.restore();
+        });
+
+        it('it should create an instance of the collection', function () {
```

Comment:
should probably also have a test to check that it works passing a collection into it already.